docusaurus/docs/quickstarts/network/hosted.md

```diff
@@ -1,168 +1,235 @@
+
+## Installation
+
+When starting out deploying an [OpenZiti Network](../../introduction/01-Introduction.mdx#overview-of-a-ziti-network), we recommend you follow
+and use the `expressInstall` function provided by the OpenZiti project. Once you're familiar with the network and
+the configuration options available you'll be better equipped to make changes.
+
+### Firewall
+
+The first issue you will need to deal with is opening some ports. A network will consist of at least one controller and 
+at least one edge router. Both of these components will require ports to be open. For the controller you will need to 
+open a range of ports through your firewall:
+
+- `8440/tcp`: Edge Controller providing router control plane
+- `8441/tcp`: Edge Controller providing client sessions
+- `8442/tcp`: Edge Router providing client connections
+- `8443/tcp`: Ziti Admin Console (ZAC) [optional]
+
+These are the arbitrary ports we'll use in this example for convenience when specifying the firewall exception as a port range.
+
+## Express Install
+
+With the firewall ports open you will now be able to source the script which provides the expressInstall function and
+execute it. You probably should consider DNS names before actually running the script. By default, the script will use the output of the
+`hostname -s` command to determine a reasonable default for the PKI that will be generated as well as the
+configuration files that are output.  The script allows you to configure these values and if you are deploying to a cloud
+provider (AWS, Azure, OCI, IBM, Digital Ocean, GCP, etc.) you will almost certainly want to use the **public** DNS name
+of your instance. You possibly  want to override the IP address to use as well, but using DNS names is superior to using
+IP addresses in case your IP changes.
+
+The quickest and easiest thing to do, is simply find your external DNS name and set it into the EXTERNAL_DNS environment
+variable. For example:
+
+```bash
+export EXTERNAL_DNS="acme.example.com"
+```
+
+:::note
+Make sure you have `jq` installed on your machine. From Debian distros that would look like:
+
+```bash
+sudo apt update && sudo apt install jq -y
+```
+
+:::
+
+Once you do that, you'll be able to
+execute these commands just as
+shown to have your
+controller and
+first edge router
+configured and ready to turn on:
+
+```bash
+# run me
+export EXTERNAL_IP="$(curl -s eth0.me)"       
+export ZITI_EDGE_CONTROLLER_IP_OVERRIDE="${EXTERNAL_IP}"
+export ZITI_EDGE_ROUTER_IP_OVERRIDE="${EXTERNAL_IP}"
+export ZITI_EDGE_CONTROLLER_HOSTNAME="${EXTERNAL_DNS}"
+export ZITI_EDGE_ROUTER_HOSTNAME="${EXTERNAL_DNS}"
+export ZITI_CTRL_PORT=8440
+export ZITI_EDGE_CONTROLLER_PORT=8441
+export ZITI_EDGE_ROUTER_PORT=8442
+```
+
+```bash
+# run expressInstall
+source /dev/stdin <<< "$(wget -qO- https://raw.githubusercontent.com/openziti/ziti/main/quickstart/docker/image/ziti-cli-functions.sh)"; expressInstall
```

Comment:
i'd be ok with slapping this file into this size and shrinking the "get ziti" command if you want too?

Reply:
Redirect added in https://github.com/openziti/ziti-doc/pull/265